.github/workflows/netcdf.yml

```diff
@@ -2,22 +2,8 @@ name: netCDF
 
 on:
   workflow_dispatch:
-  push:
-  pull_request:
-    branches: [ develop ]
-    paths-ignore:
-      - '.github/CODEOWNERS'
-      - '.github/FUNDING.yml'
-      - 'doc/**'
-      - 'release_docs/**'
-      - 'ACKNOWLEDGEMENTS'
-      - 'COPYING**'
-      - '**.md'
-
-# Using concurrency to cancel any in-progress job or run
-concurrency:
-  group: ${{ github.workflow }}-${{ github.sha || github.event.pull_request.number }}
-  cancel-in-progress: true
+  schedule:
+    - cron: "6 0 * * *"
```

Comment:
1. If this is going to be tested nightly, we should probably also test HDF-EOS5 nightly
2. These integration tests take very little time to run. Why not run them on every commit? It's a LOT easier to block bad commits than it is to track people down to revert their changes if a nightly test fails.

Reply:
Reverted to test with each PR.